reliability-engineering/pipelines/build-govsvc-docker.yml

```diff
@@ -6,6 +6,13 @@ resources:
     password: ((dockerhub-password))
     repository: govsvc/aws-ruby
 
+- name: github-aws-ruby
+  type: docker-image
+  source:
+    username: ((github-packages-username))
+    password: ((github-packages-token))
+    repository: ghcr.io/alphagov/aws-ruby
```

Comment:
think you are missing a part of the path in these...

should be:

` ghcr.io/alphagov/SOMETHING/aws-ruby`

where SOMETHING = anything you like... maybe "gds" if we think these are global ... or "autom8" or "re" if we think it's important to have a trail back to us

Reply:
That doesn't appear to be necessary. I already created the packages without adding such prefixes.

Reply:
TIL 👍